tests/foreman/cli/test_remoteexecution.py

```diff
@@ -374,20 +317,8 @@ def test_positive_run_recurring_job_with_max_iterations_by_ip(self, rex_contenth
                 'max-iteration': 2,  # just two runs
             }
         )
-
         result = JobInvocation.info({'id': invocation_command['id']})
-        try:
-            assert result['status'] == 'queued'
```

Comment:
Do you think that `queued` status you need to pass common function ? As you are only checking the status as `1` now.